requirements.txt

```diff
@@ -6,7 +6,7 @@ sphinxcontrib-mermaid==0.7.1
 sphinx-external-toc==0.2.3
 sphinx-copybutton==0.5.0
 sphinx_gitstamp==0.3.2
-beautifulsoup4==4.9.3
+beautifulsoup4==4.11.1
```

Comment:
Yep, it shouldn’t, but main was not working and I didn’t want to create two depending PRs because I hoped it would be easy walk in the park to get this in.

Reply:
Rebased to include only the changes related to the article content.